fpdf/html.py

```diff
@@ -733,16 +743,3 @@ def render_toc(self, pdf, outline):
     # Subclasses of _markupbase.ParserBase must implement this:
     def error(self, message):
         raise RuntimeError(message)
-
-
-class HTMLMixin:
```

Comment:
I think this class should be preserved, but the `write_html` method could be removed,
and a `DeprecationWarning` raised in this class `__init__()` method